doc/design/building-your-csv.md

```diff
@@ -1,3 +1,25 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```

Comment:
Nice! I think it might be valuable to add a make rule, `gen-docs`, that runs this tool against the `doc` directory. That can be a follow-up if we decide to do it though.